10_Scripts/DEADataHandling.py

```diff
@@ -406,7 +406,7 @@ def load_clearlandsat(dc, query, sensors=('ls5', 'ls7', 'ls8'), product='nbart',
 
                     # Optionally apply pixel quality mask to all observations that were not dropped in previous step
                     if mask_pixel_quality:
-                        filtered = filtered.where(good_quality)
+                        filtered = filtered.astype(np.float32).where(good_quality)
```

Comment:
should we make this configurable? maybe like `masked_dtype='float32'` with ability to overwrite with `float64|16` by users who need more control over memory/precision trade of.

Reply:
I agree, but I think the effort should be moved to `load_ard` rather than this function, as we will be retiring this one very soon. I'll create an issue for `load_ard` though

Reply:
ah,sure didn't realize this was deprecated

Reply:
Yep, this is the equivelent pull request for `load_ard`, this one was a short term fix so Juan Guerschman from CSIRO could run his analyses: https://github.com/GeoscienceAustralia/dea-notebooks/pull/358